src/bin/solang.rs

```diff
@@ -130,7 +130,13 @@ fn process_filename(
 
     let mut json_contracts = HashMap::new();
 
-    let mut f = File::open(&filename).expect("file not found");
+    let mut f = match File::open(&filename) {
+        Err(err_info) => {
+            eprintln!("{}, {:?}", err_info.to_string(), &filename);
```

Comment:
The error should look similar to other errors and warnings. So, it should be something like: 
error: cannot open ‘foo.sol’: No such file or directory

Also your commit message should say what your are changing rather than just referencing an issue. Something like "improve error handling on file system errors"

Per hyperledger dco rules you need to have your real name in the commit message. You also have two signed-off-by lines in the commit message.

Looks very good thank you very much!

Reply:
Hi Sean, I have done the following as requested.
- Changed output message
- Changed commit message
- Signed with my real name
Have a nice day, cheers !